Odata-docs/client/bulk-update-operations.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "Bulk Update Operations in OData Client"
+description: "How to perform bulk update operations in OData client"
+
+author: ElizabethOkerio
+ms.author: elokerio
+ms.date: 6/29/2023
+ms.topic: article
+ 
+---
+# Bulk Update Operations
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
+
+OData Client [v7.17.0](https://www.nuget.org/packages/Microsoft.OData.Client/) introduced support for bulk update operations.
+
+## What is a bulk update operation 
+
+A bulk update operation is an operation that allows you to perform several operations (GET, POST and PATCH) on resources of the same type and all their related resources using a single request. This can be highly efficient in situations where you need to update multiple resources and wish to avoid sending separate requests to process each resource. [Learn More](https://devblogs.microsoft.com/odata/bulk-operations-support-in-odata-web-api/).
```

Comment:
```suggestion
A bulk update operation is an operation that allows you to perform several operations (GET, POST and PATCH) on resources of the same type and all their related resources using a single request. This can be highly efficient in situations where you need to update multiple resources and wish to avoid sending separate requests to process each resource. [Learn more](https://devblogs.microsoft.com/odata/bulk-operations-support-in-odata-web-api/).
```